setup.cfg

```diff
@@ -53,6 +53,14 @@ flake8-ignore =
     docs/* ALL
     ioos_qc/tests/*.py F403 F405
 
+[flake8]
```

Comment:
So the pre-commit did find this config then. Nice! I thought this was ignored and we would need to port this to the yaml file. Much better this way!